testing/run-scripts/run_cloud.sh

```diff
@@ -52,6 +52,12 @@ then
     usage
 fi
 
+if [ "$WIPE" = true ] && [ "$UPDATE" = false ]
+then
+    echo "-u must be used when -w is used"
```

Comment:
I think they need to be separate: updates will be common, but `-w` should be required relatively rarely.